src/main/webapp/script.js

```diff
@@ -96,11 +122,14 @@ async function generateGraph() {
   let graphNodes = [];
   let graphEdges = [];
 
-  // disable buttons
+  // disable all possible input fields
```

Comment:
nit: I feel like this section is getting large enough to deserve its own method.